src/Functions/refitElec.m

```diff
@@ -1,27 +1,35 @@
-%% refitElec - refits a previously extracted corse electrode model using
-%               ptimal oblique samplingto increase accuracy and enable 
-%               automatic contact localisation. Intital refitting
-%               ("2nd pass" is followed by a final "3rd pass" to
-%               calibrate the point of origin / zero according to the detected contacts)
+function [refitReZeroedElecMod, filteredIntensity, skelScaleMm] = refitElec(initialPoly, pointCloudWorld, voxelValues, varargin)
+% refits a previously extracted corse electrode model using
+% optimal oblique sampling to increase accuracy and enable 
+% automatic contact localisation. Intital refitting
+% ("2nd pass" is followed by a final "3rd pass" to
+% calibrate the point of origin / zero according to the detected contacts)
+
+% USAGE:
+%
+%    [refitReZeroedElecMod, filteredIntensity, skelScaleMm] 
+%    = refitElec(initialPoly, pointCloudWorld, voxelValues, varargin)
 %
-% Params: initialPoly     - Dx3 double, R3 ploynomial coeffecients, 
-%         pointCloudWorld - Nx3 double, point cloud of sourounding voxels in respective coordinate system
-%         voxelValues     - Nx1 double, voxels intensitiy values (1:1 to  pointCloudWorld)
-%         varargin        - options cell
+% INPUTS: 
+%    initialPoly:           Dx3 double, R3 ploynomial coeffecients
```

Comment:
@adhusch Could you please confirm the existing descriptions and provide descriptions for the missing ones?

Reply:
confirmed.

filteredIntensity: 1D intensity profile
skelScaleMm: estimated mm per skeleton step

Reply:
Thank you very much!